entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
[shellcheck](https://www.shellcheck.net/) complains:
```
Line 35:
/build_vt.sh $GITHUB_WORKSPACE $GITHUB_WORKSPACE/build "-ftime-trace" vt
             ^-- SC2086: Double quote to prevent globbing and word splitting.
                               ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
/build_vt.sh "$GITHUB_WORKSPACE" "$GITHUB_WORKSPACE"/build "-ftime-trace" vt
 
Line 36:
vt_build_time=$(grep -oP 'real\s+\K\d+m\d+\.\d+s' $VT_BUILD_FOLDER/build_time.txt)
                                                  ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
vt_build_time=$(grep -oP 'real\s+\K\d+m\d+\.\d+s' "$VT_BUILD_FOLDER"/build_time.txt)
 
Line 39:
/build_vt.sh $GITHUB_WORKSPACE $GITHUB_WORKSPACE/build "-ftime-trace" all
             ^-- SC2086: Double quote to prevent globbing and word splitting.
                               ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
/build_vt.sh "$GITHUB_WORKSPACE" "$GITHUB_WORKSPACE"/build "-ftime-trace" all
 
Line 40:
tests_and_examples_build=$(grep -oP 'real\s+\K\d+m\d+\.\d+s' $VT_BUILD_FOLDER/build_time.txt)
>>                                                           ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
tests_and_examples_build=$(grep -oP 'real\s+\K\d+m\d+\.\d+s' "$VT_BUILD_FOLDER"/build_time.txt)
 
Line 43:
$ClangBuildTool --all $VT_BUILD_FOLDER vt-build
                      ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
$ClangBuildTool --all "$VT_BUILD_FOLDER" vt-build
 
Line 59:
    python3 /generate_graph.py -vt $vt_build_time -te $tests_and_examples_build -r $GITHUB_RUN_NUMBER
                                   ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                      ^-- SC2086: Double quote to prevent globbing and word splitting.
>>                                                                                 ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
    python3 /generate_graph.py -vt "$vt_build_time" -te "$tests_and_examples_build" -r "$GITHUB_RUN_NUMBER"
```

Reply:
Fixed